lib/recurly/hosted-fields.js

```diff
@@ -29,13 +30,55 @@ export class HostedFields extends Emitter {
     this.errors = [];
     this.initQueue = [];
     this.readyState = 0;
+    this.onTabNext = this.onTabNext.bind(this)
+    this.onTabPrevious = this.onTabPrevious.bind(this)
     this.configure(options);
     this.inject();
     this.on('hostedField:state:change', this.update.bind(this));
     this.on('bus:added', bus => {
       this.bus = bus;
       this.fields.forEach(hf => bus.add(hf));
     });
+
+    this.on('hostedField:tab:previous',this.onTabPrevious)
+    this.on('hostedField:tab:next',this.onTabNext)
+  }
+
+  onTabNext(body) {
+    let type = body.type
+    let field = this.fields.find(function(f){ return f.type === type })
+    let allFields = document.querySelectorAll("[data-recurly]")
```

Comment:
It's entirely possible for a hosted field to be the last element in the form, thus no [data-recurly] elements would follow. I think we can crawl the DOM looking for the next/prev element that is focusable (or a hosted field) and push focus to that element. jquery-ui has a `:focusable` selector method [here](https://github.com/jquery/jquery-ui/blob/89af4c292eaa5fc1c83437ca71085264dcbef34a/ui/focusable.js#L28) that implements the spec's definition of what can be focused. We can drop an analogous method into the rjs dom lib that also looks for hosted fields (which the `recurly.hostedFields` instance would tell us). It's certainly more complex, but I think it would be the best solution to make the feature work everywhere